docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p><code>Optimal Transport</code> : no need to capitalize O and T</p><p>"regularize how strictly the OT constraints are met" -&gt; "regularize a discrepancy w.r.t. marginal constraints"</p><p>"adapted Sinkhorn" -&gt; not sure what adapted means here. The original reference for this is https://papers.nips.cc/paper/2015/hash/a9eb812238f753132652ae09963a05e9-Abstract.html</p><p>Better refer to the <code>Sinkhorn</code> solver than the <code>sinkhorn</code> wrapper (maybe provide a link to doc?)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done I think